File: pset1/mario.c
#include <stdio.h>
#include <stdlib.h>
#include <ctype.h>
#include <string.h>
#include <stdbool.h>
#include <math.h>
#include <cs50.h>

int main (void)
{
  char mas[100]={0};
  int vusota=0;
  int num=0;

   do 
   {
       if (vusota <= 0 || vusota >= 24)
       {
           printf("Height ot 1 do 23: ");
           vusota=GetInt();
       }
       else
        { 
         
            vusota = GetInt();
        }  
        
         if (vusota==0)
                return 0;
                
     } while ((vusota<=0)||(vusota>23));
     


     for (int i=1; i<=vusota; i++)
     { 
        	for(int j=vusota;j>i;j--) 
        	{
        	    mas[num]=' ';
        	    num++;
        	}
    
    		for (int a=0;a<i;a++)
    		{
    		    mas[num]='#';
    		    num++;
    		}
    		
    		mas[num]=' ';
    		num++;
    		
    		mas[num]=' ';
    		num++;
    		
    		for (int a=0;a<i;a++)
    		{
    		    mas[num]='#';
    		    num++;
    		}
    
    // 		for(int i1=0;i1<strlen(mas);i1++)
    //         	printf("%c",mas[i1]);
    	        
    // 	        printf("\n");
    	puts(mas);
    	num=0;
    	
      }
   

}

File: pset6/mario.py
vusota=0
import re

while(vusota<=0) or (vusota>=24):
    if (vusota<=0) or (vusota>=24):
        str=input("Height ot 1 do 23: ")

    if re.fullmatch('[0-9]*', str) and str:
        vusota=int(str)

i=1

while i<=vusota:
    j=i

    while j<=vusota:
        print (" ",end="")
        j+=1
   
    a=0
    while a<i:
        print ("#",end="")
        a+=1
    
    print ("  ",end="")
    
    a=0
    while a<i:
        print ("#",end="")
        a+=1
    print ("")
    i+=1
        
    